plugins/watchdog/doc.go

```diff
@@ -10,3 +10,7 @@
 // times a good solution is to report to sentry and kill the worker. Ideally,
 // such bugs can be found and fixed over time.
 package watchdog
+
+import "github.com/taskcluster/taskcluster-worker/runtime/util"
+
+var debug = util.Debug("watchdog")
```

Comment:
I figured it was good place to put... the `doc.go` contains a comment that relates to the `package watchdog` statement... and this is also a once-per-package kind of thing...